### PR TITLE
Registering the broker for TSB

### PR DESCRIPTION
**Pull request by ewolinetz**
As part of setting up the TSB we need to be creating the service broker object.

Addresses: https://bugzilla.redhat.com/show_bug.cgi?id=1470623

cc @bparees @deads2k

### CODE REVIEW

File: roles/template_service_broker/tasks/install.yml [hunk 1]
```diff
@@ -42,6 +43,28 @@
...
+    # Consider using get_url or uri module rather than running curl
+    warn: no
+  register: api_health
+  until: api_health.stdout == 'ok'
+  retries: 120
+  delay: 1
+  changed_when: false
+
+- set_fact:
+    __ca_bundle: "blahblahblah"
```
Comment (bparees) [inline roles/template_service_broker/tasks/install.yml]: it's the ca bundle for the cluster (the same one it uses to create service signing certs).  not sure exactly where you can get that from though.  Presumably we're generating it during the install somewhere, it's going to live on the master.

**Comment by sdodson** [comment]
the bot was asleep on the job, all tests were green

**Pull request merged**